Auspex 1 Tweak

anything86 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Trades thermal vision in Auspex 1 for a 2-tile increase in eavesdropping range.

## Why It's Good For The Game
Thermal vision is pretty disruptive to intrigue and also doubly lore inaccurate (not only is it not a part of Auspex 1, vampires are cold). It also allows most direct targetted disciplines to be cast through walls. This aims to rectify that while also giving Auspex 1 something more lore compliant in return.

Closes #335 

## Testing Photographs and Procedure
![417b769bc39a49f9f2a3ee9bffe747c5](https://github.com/user-attachments/assets/618dc5b4-205e-4fae-b735-ff57a4801ad8)




## Changelog

:cl:
change: Auspex 1 now increases eavesdrop range in place of its thermal vision.

/:cl:

BlindPoet commented:
I think this would be great if it replaced current Auspex 5 entirely with current Auspex 1 (+ being able to hear through walls)

Also, maybe level 1 or another level could make it possible to hear people whispering in the same room as you? This PR seems like a great opportunity to make the discipline overall more fun, interactive and less clunky.

Kevak commented:
Increasing whisper hear range would be great and lore accurate.

machinePixie commented:
wouldnt say the thermal vision is disruptive really.
also view range increases are janky and awkward as it messes with the screen
i really dont like this change

zeratino commented:
not a fan of this, i don't know where this is coming from

VNLoverMay commented:
I get where you're coming from with the thermal sight - with the removal of cost from lower auspex levels, it's definitely become harder to sneak around when a bunch of vamps are just constantly able to see through walls. I don't think a 1-tile view increase is a great replacement, though. I'm not sure what *would* be, I do think this is one of the cases where we need to think outside the rulebook and come up with a novel power, but I have no idea what that should be.

shadaondiscord commented:
How about we just make every level of every discipline useless while we're at it?

Weaselburg commented:
Don't really get this.

First, the added tile range view is really fucky and bad for various reasons - it makes sprites weird, for instance. You are meant to spend your time in current sized screen and the game is built around that.

Second, how does this actually hurt intrigue? It doesn't see though cloaks and it doesn't see people actually making an attempt at hiding, like in a closet or garbage can. It does make it so movement through areas is harder but a cardboard box can counter this totally iirc, and you cant hear their conversations anyways without getting spotted.

Just solid snake that shit.

anything86 commented:
> Second, how does this actually hurt intrigue? It doesn't see though cloaks and it doesn't see people actually making an attempt at hiding, like in a closet or garbage can. It does make it so movement through areas is harder but a cardboard box can counter this totally iirc, and you cant hear their conversations anyways without getting spotted.
> 
> Just solid snake that shit.

It does see through Obfuscate (through walls) when combined with Auspex 2, which is also free. It also lets you click people through walls with most mental disciplines, affecting them, which is not something that is desired. Also I really hope you are joking about the cardboard box thing since you cannot lockpick doors  from one, and moving inside a box causes a very loud noise.

anything86 commented:
> Don't really get this.
> 
> First, the added tile range view is really fucky and bad for various reasons - it makes sprites weird, for instance. You are meant to spend your time in current sized screen and the game is built around that.

As someone who carries binoculars around all round, I don't super get this. It's also a strict range boost for firearms since they all have hitscan projectiles up to 50 tiles of distance. Lastly, if it really is something that people feel they might want to toggle off for comfort, isn't that just the actual drawback of the power as described in the books?

Regardless, I'll look into some of the other suggestions like increasing whisper range and if there is overwhelming negative feedback against the viewrange increase I'll just cut it out and replace it with whisper range.

chazzyjazzy commented:
So, couple questions:

1.) why not make it cost blood (as it used to)
2.) if you're concerned about the server's intrigue, why not go after auspex 5 which is infinitely more disruptive than thermal vision
3.) what utility and use-case does a one-tile increase in your vision give you? I don't see myself ever using this.

anything86 commented:
> 1.) why not make it cost blood (as it used to) 

Plenty of powers cost blood and do something much more substantial. I think having powers that don't cost blood and just passively increase your character's abilities is nicer, and Auspex 1 fills that role.

> 2.) if you're concerned about the server's intrigue, why not go after auspex 5 

Because other open PRs are already looking into fixing Auspex 5.

> 3.) what utility and use-case does a one-tile increase in your vision give you? I don't see myself ever using this.

Stalking someone from afar, shooting from farther with guns, noticing things a little bit earlier.

Weaselburg commented:
> It does see through Obfuscate (through walls) when combined with Auspex 2, which is also free. It also lets you click people through walls with most mental disciplines, affecting them, which is not something that is desired. Also I really hope you are joking about the cardboard box thing since you cannot lockpick doors from one, and moving inside a box causes a very loud noise.

Auspex 2 does that regardless of if Auspex 1 is engaged. It requires a really staggering amount of luck or enemy negligence to get past a high/medium pop high-traffic location (ala Tower or Bar) without going by one (one) person with Auspex 2, even if they have Auspex 1 disengaged. And this is, to be frank, only really good for stealing stuff or planting stuff - if you're trying to listen to a major conversation or meeting in progress, you either need to hide in a storage box (as I said), or have an extremely likely chance of being outed by someone in that room having auspex 2 engaged.

I know this makes osbfucate not quite as good (or less worse, depending on interpretation), but Auspex 2 will just defeat it regardless of what happens barring luck or your enemy not being there (in which you win already), and Auspex itself is a common discipline - even if Auspex 1 was nerfed, it just takes 1 Hound or Bruiser with auspex 2 engaged in the correct location (which they frequently are) to render it moot anyways.

I'm also wondering how much of this sort of intrigue you experience? Running through a building in places other people are as an obvious not-friend is pretty reckless and a bit stupid. I know for a fact someone has infiltrated the tower by pretending to be an employee, which is kind've - what you should be doing? You should be planning this stuff out and not relying on one singular power to perform sneakiness. I would like for Obfuscate not to be seen through walls, too, but while the one-tile increase would be nice _in theory_, and I might not immediately riot upon testing it, it really would mess with sprites and perception - because again, the game is really build for the current screen. Maybe it'd work? I kinda doubt it, but if Thermal Eyes being removed is the will of the Gods then that + adding a range where you can hear whispers + extending hearing would at least be a replacement worth testing. Maybe. I'd miss thermal eyes heavily and especially Auspex 4 mindbeams-through-walls.

I wasn't aware the sound of a cardboard box moving carried through walls but if it does the simple solution is to reduce it or make it not carry so much so that people can sneak better in it. This should be done regardless and I'd do it myself if me touching code didn't result in server implosions 100% of the time, because it's funny and clever and also counterable with your two eyes. 

I can't speak for what's wanted for mental discs.

anything86 commented:
> (snip)

How does that relate to the removal of thermal vision? Auspex 2 should not be seeing auras non-stop through the walls, either, but that is an issue to be fixed in an Auspex 2 PR.

I think you're greatly mistaking the scope of this PR. And if another Auspex power is doing the exact same thing, what is the issue with removing one source of the problem here?

anything86 commented:
Right, some quick livetesting shows that Auspex 2 does not currently let you see people through walls by itself. I henceforth have no idea what that person was talking about.

anything86 commented:
Removed the viewrange increase, added a 2 tile increase to eavesdropping range tied to a trait.

Enabling hearing through walls would require refactoring quite a bit of the say/hearing code, so I personally think that is best left for a more ambitious PR.

Weaselburg commented:
By "correct location" I meant within actual normal view, sorry if that wasn't clear enough. Auspex 2 indeed never lets you see through walls on it's own.

machinePixie commented:
whisper hearing isnt really much of a replacement to be honest, this just feels like a PR to remove something you dont like and leave it gutted at this point, honestly. no offense.
id rather not have a half-hearted effort of taking something out and leaving a cardboard cutout with an IOU on it that'll never get paid.

anything86 commented:
> whisper hearing isnt really much of a replacement to be honest, this just feels like a PR to remove something you dont like and leave it gutted at this point, honestly. no offense. id rather not have a half-hearted effort of taking something out and leaving a cardboard cutout with an IOU on it that'll never get paid.

It's a level one power. Permanent thermal vision that allows you to fling all targetted disciplines across walls shouldn't be in its description as a level one power. Feel free to name another level 1 power that is more useful.

Weaselburg commented:
I don't think it's fair to compare Temporis 1 to any of the other 1 dots given that it's _intentionally_ useless due to how temporis ramps up crazily in later dots. It's also lore-accurate, though Temp 1 also lets you sense distrubances in the time force (which obviously can't be done very well here).

Celerity 1 boosts your speed so you can kite out of distance, basically making you immune to melee to someone without extra speed. Obsfucate 1 makes you invisible if you stand still to anyone not with auspex 2 (iirc). Potence 1 increases your melee damage (essential for an aspiring bruiser), dom 1 has your get-out-of-jail-free card and VoG (which has the funny stuns as well).

All the common disc 1 dots are _very_ good. Some are a little better or worse than the others, but they're all very useful. Auspex 1 coordinates with mentals very well but Celerity does the same with actual physical combat.

anything86 commented:
> (snip)

Celerity 1, Obfuscate 1, Potence 1 are all completely obsoleted by later dots. There is no reason to ever use Obfuscate 1 over Obfuscate 2. 3 out of 4 examples you've given are worse than just being comparatively worse to Auspex 1, they are strictly, objectively, downgrades to other powers in their same disciplines.

Meanwhile Auspex 1 stacks on top of anything for free. It is a free bonus that is always active. **It should not give you wallhacks as a 1 dot power for free permanently. Even the concept of a 'wallhack' flies in the face of tabletop rules for most disciplines like Dominate and Presence.**

Weaselburg commented:
I was not arguing in favor of wallhacks, just that other 1 dots were also pretty damn good and people ignore it because they either move up to other dots immediately or discard them as silly.

Whisper range is helpful and pretty decent, honestly. 3 feels a bit weak but it's not like that can't be buffed later. I'll miss telepathy wall convos immensely but I think it's worth it to stop dominate-through-walls.

Lutowski commented:
![image](https://github.com/user-attachments/assets/a61586a2-d78f-4e20-8584-faf6498fc391)

i think that clever combos with auspex SHOULDN'T be punished actually
